server/controllers/request.js

```diff
@@ -0,0 +1,90 @@
+const Request = require("../models/Request");
+const asyncHandler = require("express-async-handler");
+
+// @route GET /request/all
+// @desc List of requests for logged in user
+// @access Private
+exports.getRequests = asyncHandler(async (req, res, next) => {
+  const userId = req.user.id;
+  const ownerRequests = await Request.find({ owner: userId });
```

Comment:
Why don't we simplify this as opposed to querying the database twice: `Request.find({ $or: [{ user: req.user.id }, { sitter: userProfile.profile._id }] })` 😄